deltachat-ffi/deltachat.h

```diff
@@ -3558,6 +3616,42 @@ char*           dc_msg_get_filename           (const dc_msg_t* msg);
 char*           dc_msg_get_filemime           (const dc_msg_t* msg);
 
 
+/**
+ * Return file from inside an webxdc message.
+ *
+ * @memberof dc_msg_t
+ * @param msg The webxdc instance.
+ * @param filename The name inside the archive,
+ *     can be given as a absolute path (`/file.png`)
+ *     or as a relative path (`file.png`, no leading slash)
+ * @param ret_bytes Pointer to a size_t. The size of the blob will be written here.
+ * @return The blob must be released using dc_str_unref() after usage.
+ *     NULL if there is no such file in the archive or on errors.
+ */
+char*             dc_msg_get_webxdc_blob      (const dc_msg_t* msg, const char* filename, size_t* ret_bytes);
+
+
+/**
+ * Get info a webxdc message, in JSON format.
```

Comment:
get info of* a webxdc message?

Reply:
i think "from" is better, but sth. is missing indeed :)